src/protagonist/DLCS.Repository.Tests/DLCS.Repository.Tests.csproj

```diff
@@ -28,6 +28,7 @@
     </ItemGroup>
 
     <ItemGroup>
+      <ProjectReference Include="..\API\API.csproj" />

```

Comment:
Why? Doesn't seem right

Reply:
think it snuck in from when I was working on this - will remove